pkgs/data/misc/hackage/default.nix

```diff
@@ -1,6 +1,7 @@
+# Hackage database snapshot, used by maintainers/scripts/regenerate-hackage-packages.sh
+# and callHackage
 { fetchurl }:
-
-fetchurl {
-  url = "https://github.com/commercialhaskell/all-cabal-hashes/archive/d202e2aff06500ede787ed63544476f6d41e9eb7.tar.gz";
-  sha256 = "00hmclrhr3a2h9vshsl909g0zgymlamx491lkhwr5kgb3qx9sfh2";
-}
+let
+  pin = builtins.fromJSON (builtins.readFile ./pin.json);
+in
+  fetchurl { inherit (pin) url sha256; }
```

Comment:
Can we add the update script via `passthru` here?

Reply:
Can you enlighten me as to who or what does actually use the passthru update script?

I am a bit sceptical because at least fetchgit didn‘t accept a passthru attribute.

Reply:
`fetchurl` accepts `passthru`. IIRC some automation uses it.

Reply:
Okay, done.

`nix-shell maintainers/scripts/update.nix --argstr package all-cabal-hashes` is an alias for our script now.^^